src/UI/Factory.php

```diff
@@ -336,4 +336,23 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A divider marks a thematic change in a sequence of other components.
+	 *   composition: >
+	 *     A divider consists of a horizontal ruler which may be accompanied by a label.
+	 *
+	 * rules:
+	 *   usage:
+	 *     1: >
+	 *       Dividers MUST only be used in in container components that render
+	 *       a sequence of items.
+	 * ---
+	 *
+	 * @return \ILIAS\UI\Component\Divider\Factory
+	 **/
+	public function divider();
```

Comment:
This seems to be too generic for me, since it's only a horizontal divider, but there could be vertical as well.